django/applications/catmaid/control/annotation.py

```diff
@@ -1119,9 +1118,10 @@ def create_annotation_query(project_id, param_dict):
     return annotation_query
 
 
-def generate_annotation_intersection_query(project_id, annotations):
+def generate_annotation_intersection_query(project_id:Union[int,str], annotations) -> Optional[str]:
+    # XXX This function is likely unused.
```

Comment:
Yes, looks like it and looking at the history it seems like this function wasn't ever used. In fact it might have been the first attempt to implement the function below (`generate_co_annotation_query()`). The function should be removed.